gnucash/import-export/import-main-matcher.c

```diff
@@ -1394,6 +1394,9 @@ gnc_gen_trans_view_popup_menu (GtkTreeView *treeview,
     gtk_menu_popup_at_pointer (GTK_MENU(menu), (GdkEvent*)event);
 
     g_list_free_full (selected_rows, (GDestroyNotify)gtk_tree_path_free);
+
+    g_object_take_ref (G_OBJECT(menu));
+    g_object_unref (G_OBJECT(menu));
```

Comment:
Any more `g_object_unref` fails. So, I presume this is the right approach to fix this leaking `menu` object.